erc721-meta-transaction/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "lib": [
+      "ES2021",
+      "DOM"
+    ],
+    "module": "commonjs",
+    "outDir": "dist",
+    "strict": true,
+    "target": "ES2015"
+  },
+  "include": [
+    "src"
+  ]
+}
```

Comment:
no idea if this is good or not, maybe double check with ilya / david

Reply:
Checked with @davidzwfu and he said it looks fine